src/components/CardEditor/CardEditForm/CardEditForm.test.jsx

```diff
@@ -90,4 +97,58 @@ describe('CardEditForm', () => {
       expect(mockSetError).toBeCalledWith('1234 is not valid JSON');
     });
   });
+  describe('hideCardPropertiesForEditor', () => {
```

Comment:
can you add a test for `series` and `attributes` both being hidden at the same time?

Reply:
This case wouldn't happen in our current card schemas.